soh/soh/Enhancements/randomizer/3drando/hint_list.cpp

```diff
@@ -2253,10 +2253,15 @@ void StaticData::HintTable_Init() {
                                                                {QM_GREEN, QM_RED}));
 
     hintTextTable[RHT_SARIA_SONG_HINT] = HintText(CustomMessage("Did you feel the #surge of magic# recently? A mysterious bird told me it came from #[[1]]#.^You should check that place out, @!\x0B",
```

Comment:
I don't remember the reason of that control code being there, but if you remove it in German and French, better to remove it in English too :P
```suggestion
    hintTextTable[RHT_SARIA_SONG_HINT] = HintText(CustomMessage("Did you feel the #surge of magic# recently? A mysterious bird told me it came from #[[1]]#.^You should check that place out, @!",
```

Reply:
mm I should revert removing that, somehow got removed in copy/pasting